### PR TITLE
Horizontaly aligned components

### PR DESCRIPTION
**Pull request by lorang92**
https://github.com/Altinn/altinn-studio/issues/1515

### CODE REVIEW

**Review by lvbachmann — APPROVED** [review]
Ready for merge when test for issue is complete 👍

File: content/altinn-studio/app-creation/ui-editor/layout-style/_index.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+- Kort svar (Input)
+- Langt svar (TextArea)
+- Filopplaster (FileUpload)
+- Nedtrekksliste (Dropdown)
+- Datovelger (Datepicker)
+
+
+Eksempelet over vil gi følgende output:
+
+![InnerGrid eksempel output](inner-grid.PNG "InnerGrid eksempel output")
```
Comment (altinnadmin) [inline content/altinn-studio/app-creation/ui-editor/layout-style/_index.md:132]: It's a mystery to me why extensions for image-files surprisingly often ends up in terrible upper case. 😃 What kind of tool did you use for images? All extensions should be made lowecase.

File: content/altinn-studio/app-creation/ui-editor/layout-style/_index.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+{{%notice warning%}}
+Dette er helt ny funksjonalitet. Oppsett må gjøres manuelt inntil videre. Støtte for oppsett via Altinn Studio kommer snart.
+{{%/notice%}}
+
+## Sidestilte komponenter
+
+Det er mulig å sidestille komponenter i skjema. Denne funksjonaliteten baserer seg på grid systemet til [Material-UI](https://material-ui.com/components/grid/), og vil også være kjent om man er kjent med grid-systemet til boostrap.
+Grid systemet baserer seg på en oppbygning av containere i 12 deler, hvor man så kan tildele barn (items) av en grid-container en gitt andel av denne bredden. Material-UI tilbyr også muligheten til å dynamisk endre hvor stor andel av skjermen man ønsker å oppta basert på hvilken skjermstørrelse sluttbrukeren har.
+Følgende størrelser med tilhørende breakpoints er definert:
+```
```
Comment (altinnadmin) [inline content/altinn-studio/app-creation/ui-editor/layout-style/_index.md:17]: Add txt type, or just use a simple markdown list?

File: content/altinn-studio/app-creation/ui-editor/layout-style/_index.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+        }
+      }
+}
+
+```
+
+I komponentene over er størrelsen satt til 6 (altså halv bredde) for størrelse `xs`. Siden `xs` er den minste størrelsen tilgjengelig vil dette gjelde fra størrelse `xs` og opp, altså fra `xs` til `xl`.
+Oppsettet over vil gi følgende resultat:
+
+![Halv bredde på skjema](halv-bredde.PNG "Halv bredde på skjema")
```
Comment (altinnadmin) [inline content/altinn-studio/app-creation/ui-editor/layout-style/_index.md:52]: See comment below about uppercase extensions.

**Review by altinnadmin — APPROVED** [review]
Very nice, a few nitpicks to fix before merging :)

File: content/altinn-studio/app-creation/ui-editor/layout-style/_index.md (hunk 2 above)
Comment (lorang92) [reply content/altinn-studio/app-creation/ui-editor/layout-style/_index.md]: Agreed, fixed with a markdown list.

File: content/altinn-studio/app-creation/ui-editor/layout-style/_index.md (hunk 1 above)
Comment (lorang92) [reply content/altinn-studio/app-creation/ui-editor/layout-style/_index.md]: Snipping tool 😠 fixed.

Comment (altinnadmin) [reply content/altinn-studio/app-creation/ui-editor/layout-style/_index.md]: Ahhhh, thanks, nice to know! :)

Comment (lorang92) [reply content/altinn-studio/app-creation/ui-editor/layout-style/_index.md]: Interesting. git (or possibly vs-code) does not recognize changing file extensions from lowercase to uppercase as an actual change to the file (guessing this is related to windows being case insensitive for file paths).

Looking into how I can fix this 🤓

Comment (lorang92) [reply content/altinn-studio/app-creation/ui-editor/layout-style/_index.md]: `git config core.ignorecase false` did the trick. Fixed.

**Pull request merged**